offline/cable_parameters.F90

```diff
@@ -219,16 +219,14 @@ SUBROUTINE read_gridinfo(nlon, nlat, npatch)
     ok = NF90_OPEN(trim(filename%type), 0, ncid)
     IF (ok /= NF90_NOERR) CALL nc_abort(ok, 'Error opening grid info file.')
 
-    ok = NF90_INQ_DIMID(ncid, 'longitude', xID)
-    IF (ok /= NF90_NOERR) ok = NF90_INQ_DIMID(ncid, 'x', xID)
-    IF (ok /= NF90_NOERR) CALL nc_abort(ok, 'Error inquiring x dimension.')
+    xID = longitude_dimid(ncid)
```

Comment:
Use the suggestion function `read_dimid` with `ncid` and possible names defined as parameter in the same module.